lib/bb/generate_report_request_form.rb

```diff
@@ -25,15 +25,22 @@ class GenerateReportRequestForm < Common::Form
     # MHV error is preferable.
     # validates :from_date, date: { before: :to_date, message: 'must be before to date' }
     validates :data_classes, presence: true
-    validate  :data_classes_belongs_to_eligible_data_classes
+    # TODO: temporary hack to make eligible data classes work
```

Comment:
The TODO comments are not very helpful for the next person that comes along in describing why this is a hack and what the correct resolution is (presumably providing an eligible data classes API method and letting frontend render the form accordingly). It's not even that big of a hack, filtering the provided data classes according to what's eligible. Please clarify comments.